src/transform2d.rs

```diff
@@ -18,51 +18,51 @@ use trig::Trig;
 use std::fmt;
 
 define_matrix! {
```

Comment:
It's the same macro used with points and other euclid types, so I think it's fine to leave it as matrix.